.github/workflows/enterprise-quality-gate.yml

```
@@ -53,93 +54,149 @@ jobs:
+    except ImportError:
+        pytest.skip("App module not importable")
+
+
+def test_sonar_config():
+    """Verify SonarQube configuration exists."""
+    assert os.path.exists('sonar-project.properties')
+    
+    # Read the sonar-project.properties file
+    with open('sonar-project.properties', 'r') as f:
+        content = f.read()
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)